webapp/src/index.tsx

```diff
@@ -5,14 +5,16 @@ import * as React from 'react';
 
 import {Channel} from 'mattermost-redux/types/channels';
 import {Post} from 'mattermost-redux/types/posts';
-
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
```

Comment:
Can we add the blank line back in here, to separate the imports from `mattermost-redux` and local imports?

Reply:
Sure